components/notifier/src/quality_time_notifier.py

```diff
@@ -24,8 +24,9 @@ async def notify(log_level: int = None) -> NoReturn:
         f"{os.environ.get('SERVER_PORT', '5001')}/api/{api_version}/reports"
     )
     data_model = await retrieve_data_model(api_version)
-    most_recent_measurement_seen = datetime.max.isoformat()
+    most_recent_measurement_seen = datetime.max.isoformat() + "+00:00"
     outbox = Outbox()
+    notable = NotableEvents(data_model)
```

Comment:
"notable" is an adjective. Class instance names should be a noun. It's a bit hard in this case, but maybe "notification_finder"?